linters.go

```diff
@@ -1,30 +1,44 @@
 package main
 
 import (
-	"encoding/json"
 	"fmt"
 	"os"
 	"os/exec"
 	"regexp"
+	"sort"
 	"strings"
 
 	"gopkg.in/alecthomas/kingpin.v3-unstable"
 )
 
+type LinterConfig struct {
+	Name              string
+	Command           string
+	Pattern           string
+	InstallFrom       string
+	PartitionStrategy partitionStrategy
+	IsSlow            bool
```

Comment:
sounds good, will swap to `IsFast`

a common interface for linters sounds like an interesting idea